src/App.js

```diff
@@ -18,7 +18,7 @@ class App extends Component {
       <CategoryProvider>
         <CategoryContext.Consumer>
           {categoryContext => (
-            <Router>
+            <Router basename="/fsd-spike">
```

Comment:
@danieljauch This looks right to me 👍

Reply:
Yeah, I didn't even know this was a thing until @mischa-s pointed it out. It will need to change when the repo changes, so I'm going to put it into `.env`